chalicelib/check_setup.json

```diff
@@ -1227,5 +1227,16 @@
                 }
             }
         }
+    },
+    "elastic_search_space" : {
+        "title": "ES Disk Space Check",
+        "group": "Elasticsearch checks",
+        "schedule": {
+            "morning_checks": {
+                "all": {
+                    "kwargs": {}
```

Comment:
This should be `"kwargs" : {"primary": true}`. The Foursight UI only displays the "primary" result, which is done so that you can test checks without overwriting

Reply:
Added!